src/components/MarkdownProvider/components/BestPractice.tsx

```diff
@@ -44,6 +45,8 @@ const StyledImg = styled.img`
   border-bottom: none;
   border-top-left-radius: ${p => p.theme.borderRadii.md};
   border-top-right-radius: ${p => p.theme.borderRadii.md};
+  padding: ${p => p.theme.space.md};
+  object-fit: cover;
```

Comment:
I noticed that `object-fit` is listed as unsupported for IE11. Does the image fit okay in IE11? 

https://caniuse.com/#feat=object-fit

Reply:
Similar to https://github.com/zendeskgarden/react-components/blob/5bebfa80a1155452eed1aae61f2041f9efbd4691/packages/avatars/src/styled/StyledAvatar.ts#L262, we want to optimize for the browsers that can benefit. In general, image skew will be addressed when the designers resize taking padding into account.

Reply:
...but scaling on IE11 can definitely be improved. Fixing that now.